taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -124,3 +124,32 @@ def fill_email_data(config, tasks):
             email["subject"] = email["subject"].format(**format_kwargs)
 
         yield task
+
+
+@transforms.add
+def filter_ytp_tests(config, tasks):
+    """Youtube-playback AV1 and VP9 tests should run on P2 only"""
+    for task in tasks:
+        test_platform = task['treeherder']['platform']
+        test_name = task['name']
+        if "AV1" in test_name or "VP9" in test_name:
+            if 'android-hw-p2' in test_platform:
+                yield task
+            else:
+                continue
+
+        # Bug 1668537 The following tests are failing intermittently
+        # and are disabled until the error is fixed
+        if "WIDEVINE-H264" in test_name or "WIDEVINE-H264-HFR" in test_name:
```

Comment:
Why is this capitalized? If there are issues with capitalization you should compare a lower case string to `test_name.lower()`. Also, I don't think we need a split below because it doesn't work on any of the devices, so we should change all that to a continue.